.eleventy.js

```diff
@@ -93,6 +94,46 @@ module.exports = function (eleventyConfig) {
     return moment(date).format(format);
   });
 
+   // Token filters
+  eleventyConfig.addFilter("contrast", function (value, b = "#FFF") {
+    let contrast = chroma.contrast(value, b);
+    return Number.parseFloat(contrast).toFixed(2);
```

Comment:
I know this is being picky, but can we round these values down. That way if someone runs it through webaim like mentioned on our colour page, the values will match.

Reply:
[toFixed](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed), already rounds down I believe. Did you want us to get rid of the zeros?

Reply:
Zeros should be fine.

toFixed just rounds it, doesn't round down. An example would be `--gcds-color-grayscale-100` lists contrast as 1.42 but webaim says 1.41. Just want to be consistent with them.

Reply:
Perfect. I've changed it to floor the value.